### PR TITLE
Implement Edit.LineOpenAbove and Edit.LineOpenLineBelow commands for XML doc comment generation

### LINKED ISSUE #2091: Incorrect behavior of ENTER in XML documentation comment when whitespace is to the right of the editor caret
**Repro Steps:**

* Enter the following code, and position the caret at the location indicated by `|`. *Note that the code block is indented.*
```C#
            /// <summary>
            /// foo| bar
            /// </summary>
            public void Foo() { }
```

* Press Enter

**Expected Result:**
```C#
        /// <summary>
        /// foo
        /// |bar
        /// </summary>
        public void Foo() { }
```

**Actual Result:**
```C#
        /// <summary>
        /// foo
       /// | bar
        /// </summary>
        public void Foo() { }
```

### PR DESCRIPTION
**Pull request by DustinCampbell**
Fixes #2091.

Reviewers: @Pilchie, @rchande, @dpoeschl, @jasonmalinowski, @davkean, @jmarolf, @CyrusNajmabadi, @brettfo

### CODE REVIEW

**Comment by DustinCampbell** [comment]
Also, @balajikris

**Comment by Pilchie** [comment]
:+1:

File: src/EditorFeatures/Core/Implementation/DocumentationComments/AbstractDocumentationCommentCommandHandler.cs [hunk 1]
```diff
@@ -587,5 +569,143 @@ public void ExecuteCommand(InsertCommentCommandArgs args, Action nextHandler)
                     }
                 });
         }
+
+        public CommandState GetCommandState(OpenLineAboveCommandArgs args, Func<CommandState> nextHandler)
+        {
+            return nextHandler();
+        }
+
+        public void ExecuteCommand(OpenLineAboveCommandArgs args, Action nextHandler)
```
Comment (dpoeschl) [inline src/EditorFeatures/Core/Implementation/DocumentationComments/AbstractDocumentationCommentCommandHandler.cs]: Consider de-duping the implementation here with that of OpenLineBelow's ExecuteCommand below.

**Comment by dpoeschl** [comment]
:+1:

**Pull request merged**